tests/package.lisp

```diff
@@ -24,14 +24,16 @@
   (:documentation "Tests for the COALTON system, written in Coalton.")
   (:use #:coalton-testing)
   (:local-nicknames
+   (#:fun  #:coalton-library/functions)
```

Comment:
`coalton-library/functions` should be used already

Reply:
`coalton-testing` use-reexports `coalton-prelude` which use-reexports `coalton-library/functions`.